src/index.html

```diff
@@ -2,16 +2,122 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form class="form" action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+     onsubmit="onSubmit(Submit)"
+     method="post">
+      <fieldset class="form-field1">
+        <legend>Personal information</legend>
+        <label class="formlabel1" for="surname">
+          Surname:
+          <input id="surname" type="surname">
+        </label>
+        <label class="label" for="name">
+          Name:
+          <input id="name" type="text">
+        </label>
+        <label class="label"  for="number">
+          How old are You ?
+          <input class="form-number"
+           type="number"
+           id="number"
+           value="12"
+           min="0"
+           max="100"
+          >
+        </label>
+        <label class="label"  for="formdate"> Full date of birth
+          <input id="formdate" type="date"
+           name="formdate"
+          >
+        </label>
+        <label class="label2" for="accept">
+          I accept the term of the agreement
+          <input id="accept" type="checkbox">
+        </label>
+      </fieldset>
+      <fieldset class="form-field">
+        <legend>Registration</legend>
+        <label class="label" for="email">
+           E-mail:
+          <input
+           id="email"
+           type="email"
+           placeholder="email@example.com"
+          >
+        </label>
+        <label class="label" for="password">
+          Password: <input id="password" type="password">
+        </label>
+      </fieldset>
+      <fieldset class="form-field">
+        <legend>An interesting fact about you!</legend>
+        <label
+          for="Cats"
+          class="label"
+          > Do you love cats?
+          <input
+           id="Cats"
+           type="radio"
+           name="Cats"
+          > Yes
+          <input type="radio" name="Answer">
+          No
```

Comment:
```suggestion
          <input
            type="radio"
            name="сats"
          >
          Yes
          <input
            type="radio"
            name="сats"
          >
          No
```
радіо інпути мають мати однакове name інакше вони не зв'язані і можна обрати обидва варіанти одночасно